README.md

```diff
@@ -34,10 +34,12 @@
 - [License](#license)
 
 > **⚠️ Warning ⚠️**
-> The project has recently been renamed from QHub to Nebari. If your deployment is still managed by `qhub`, performing an in place upgrade will **IRREVOCABLY BREAK** your deployment.
-> This will cause you to lose any data stored on the platform, including but not limited to, NFS (file system) data,
-> `conda-store` environments, Keycloak users and groups, etc.
-> Make sure to [back up your data before attempting an upgrade](https://www.nebari.dev/docs/how-tos/manual-backup).
+> The `2023.9.1` release includes the initial implementation of a [Pluggy-based](https://pluggy.readthedocs.io/en/stable/) extension mechanism, for more details refer [here]().
+> This version also fully deprecates CDS Dashboards as it is no longer compatible with the newer versions of JupyterHub.
```

Comment:
*deprecate* implies it's still available but not recommended for use and will be shut off later. Would think it'd be more accurate to just say it's been removed in this release and refer to the previous deprecation announcement.